website/docs/r/licensemanager_association.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_licensemanager_association"
+sidebar_current: "docs-aws-resource-licensemanager-license-configuration"
+description: |-
+  Provides a License Manager association resource.
+---
+
+# aws_licensemanager_association
+
+Provides a License Manager association.
+
+~> **Note:** License configurations can also be associated with launch templates by specifying the `license_specifications` block for an `aws_launch_template`.
+
+## Example Usage
+
+```hcl
+resource "aws_licensemanager_license_configuration" "example" {
+  name                  = "Example"
+  license_counting_type = "Instance"
+}
+
+resource "aws_licensemanager_association" "example" {
+  license_configuration_arn = "${aws_licensemanager_license_configuration.example.arn}"
+  resource_arn              = ...
```

Comment:
Nit: We should show an example reference here and/or update the `resource_arn` argument documentation below to include types of references supported

```suggestion
  resource_arn              = "${aws_instance.example.arn}"
```